lib/api/route/realm.dart

```diff
@@ -51,7 +51,7 @@ class GetServerSettingsResult {
   final String realmName;
   final String realmIcon;
   final String realmDescription;
-  final bool? realmWebPublicAccessEnabled; // TODO(server-5)
+  final bool realmWebPublicAccessEnabled;
```

Comment:
```
TODO api: Mark GetServerSettingsResult.realmWebPublicAccessEnabled as required, relying on server 5+, FL 116+

See https://github.com/zulip/zulip-flutter/pull/904#discussion_r1747943940.

See "Feature level 116" from Zulip API changelog:

   https://zulip.com/api/changelog

Signed-off-by: Zixuan James Li <zixuan@zulip.com>
```

as in a previous comment of mine: this is fine; no interaction with disallow-ancient-servers